01-shapefile-attributes.Rmd

```diff
@@ -147,34 +148,21 @@ extent(point_HARV)
 point_HARV
 ```
 
-
 ##About Shapefile Attributes
 Shapefiles often contain an associated database or spreadsheet of values called
-`attributes` that describe the vector features in the shapefile. You can think
+**attributes** that describe the vector features in the shapefile. You can think
 of this like a spreadsheet with rows and columns. Each column in the spreadsheet
-is an individual `attribute` that describes an object (sometimes called
-variables). Shapefile `attributes` include measurements that correspond to the
+is an individual **attribute** that describes an object (sometimes called a
+variable). Shapefile attributes include measurements that correspond to the
```

Comment:
sure - i can see what that would be confusing. Why don't we just remove the part in () about variable? it's probably just confusing to add here in general. Does that work?